test/test_builders.py

```diff
@@ -67,6 +67,15 @@ def mock_zarr_store(entity_path, mocker):
             z['obs/predicted.ASCT.celltype'] = True  # only checked for membership in zarr group
         elif 'predicted-label' in entity_path.name:
             z['obs/predicted_label'] = True  # only checked for membership in zarr group
+    if 'marker' in entity_path.name:
+        obs = z.create_group('obs')
+        obs.attrs.__setitem__('encoding-version', '0.1.0')
+        var = z.create_group('var')
+        var.attrs.__setitem__('_index', 'index')
+        var.array('index', ['ENSG00000139618', 'ENSG00000139619', 'ENSG00000139620'])
```

Comment:
Didn't even know you could do this.  I think sticking to the tutorial patterns is better: https://zarr.readthedocs.io/en/stable/tutorial.html
i.e., 
```suggestion
        var['index'] = zarr.array(['ENSG00000139618', 'ENSG00000139619', 'ENSG00000139620'])
```
i think?